core/src/test/java/ch/cyberduck/core/LoginCallbackFactoryTest.java

```diff
@@ -32,4 +40,60 @@ public void invoke(final MainAction runnable, final boolean wait) {
             }
         }));
     }
+
+    @Test
+    @Ignore
```

Comment:
Ignored, because either intended, or exceptional.
If exceptional, this should be fixed, otherwise keep as ignored, to make communicate, that this behavior is intended.

Reply:
Can changes in f92b8a300820b481cc6f49a64d18f0ba85379f7d be reverted with removed constructor caching in f2f8abb2ce4e6a066719ab7c253454975fb88e96?

Reply:
Don't know what the performance implication here is. Reflection is slow.

Reply:
These callbacks are not requested frequently that it would have any noticeable performance implication.

Reply:
The change in f92b8a3 can't be reverted, as the LoginCallback requires a WindowController in its constructor, which the BaseController (which is registered in the DI Container) is not a subtype of, thus DisabledLoginCallback would be used for the profiles fetcher (and a warning logged).

Reply:
Loading profiles will require no login and thus it is not a problem if `DisabledLoginCallback` is used.

Reply:
Will check what I can do to get f92b8a3 reverted.

Reply:
Can the log.warn in the callback factories be gated behind log.isDebugEnabled?

Reply:
The log statement doesn't do any harm.